src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -133,6 +142,42 @@ export default class BraintreeHostedForm {
         }
     }
 
+    async tokenizeWith3DSRegulationCheck(billingAddress: Address, merchantAccountId: string): Promise<Pick<BraintreeHostedFieldsTokenizePayload, 'authenticationInsight'> & NonceInstrument> {
+        if (!this._cardFields) {
+            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+        }
+
+        try {
+            const { authenticationInsight, nonce } = await this._cardFields.tokenize(omitBy({
```

Comment:
Can we please break this into multiple lines, it can be a bit harder to read in this current state.

Reply:
Ok, I will break this into multiple lines

Reply:
Fixed